pywinauto/controls/common_controls.py

```diff
@@ -3388,6 +3388,58 @@ def set_view(self, viewType):
         if res == 0:
             raise RuntimeError('Failed to set view in Calendar')
 
+    #----------------------------------------------------------------
+    def set_day_states(self, month_states):
+        """Sets the day states for all months that are currently visible within a month calendar control"""

```

Comment:
Phrase `within a month calendar control` is obsolete and it makes the doc string too long.